test/lib/ufe/ufeTestUtils/mayaUtils.py

```diff
@@ -20,6 +20,7 @@
     Helper functions regarding Maya that will be used throughout the test.
 """
 
+from __future__ import print_function
```

Comment:
For the way `print()` is used below, I don't think you need this. It was only needed in `build.py` because of the parameter-style usage: `print("Message:", msg)`.

Reply:
Hey Matt, I am not sure if I see the connection with build.py.

This is a separate python file and without this change almost all ufe tests fail. 

Perhaps there is something I don't see?

Reply:
I meant that I don't think you should need this `import` statement. You *do* need the changes below to use `print()` as a function.

I didn't mean to imply that there was a connection to `build.py`. I just meant that because of the way `build.py` uses `print()`, you *do* need the `from __future__ import print_function` statement there.

Reply:
Ahhh I see now. My apologies for the confusion.

> For the way print() is used below, I don't think you need this

which print() are you referring to?  There are two in here:

```
print(sys.exc_info()[1])
print("Unable to load %s" % pluginName)
```

Reply:
I was trying to highlight the difference in the single parameter version of `print()` vs. the multiple-parameter version, and how the behavior there is different between Python 2 and Python 3.

Single vs. Multi-parameter In Python 2:
```
> python --version
Python 2.7.10
> python -c "msg = 'This is a string'; print('Message: %s' % msg)"
Message: This is a string
> python -c "msg = 'This is a string'; print('Message:', msg)"
('Message:', 'This is a string')
```

Single vs. Multi-parameter In Python 3:
```
> python3 --version
Python 3.7.7
> python3 -c "msg = 'This is a string'; print('Message: %s' % msg)"
Message: This is a string
> python3 -c "msg = 'This is a string'; print('Message:', msg)"
Message: This is a string
```

Notice that the multi-parameter version in Python 2 prints tuple-like output, which we don't want. That's what requires importing from the future:

In Python 2:
```
> python -c "from __future__ import print_function; msg = 'This is a string'; print('Message:', msg)"
Message: This is a string
```

So since `build.py` is using the multi-parameter version, we need to import from the future to get the right output. Since `mayaUtils.py` is not, we should **not** need to import from the future.

Reply:
@mattyjams  Ughh, I didn't know this.  Thanks, you convinced me.  

Done. See d10a527af2a42983e9d76263c3528ee1236d8792